### ISSUE TITLE
Can't create combined CSS file with HTTPS sites

### ISSUE THREAD

**Issue by philippreiner** [flipzoom/ProcessWire-AIOM-All-In-One-Minify#7]
Occured on ProcessWire 2.3.8 (also tested with 2.3.5) dev using the latest (2.2.1) AIOM module.

I'm forcing HTTPS for the whole site, by setting the URL on the home template to "HTTPs only". After doing this, the creation of a new combined file fails. If I switch back to HTTP and open the site via the http:// url to generate the combined file  it works. Enabling HTTPs afterwards without any changes to the CSS/JS works too.

To summarize:
HTTPS, new CSS -> AIOM fails
HTTP, new CSS -> AIOM works
HTTPS after HTTP, no changes to CSS -> AIOM works until CSS/JS Change.

I'm getting this error:
Exception: The combined css cache file could not be created. (in /home/httpd/vhosts/***/site/modules/AllInOneMinify/AllInOneMinify.module line 517)

**Comment by flipzoom** [comment]
Hi Philipp, 

I can not reproduce the error unfortunately. Both with version 2.3 (stable) to 2.3.8. I think it's a problem with the rights. What rights do you have in default-configuration (in site/config.php: chmodDir and chmodFile)? Is PHP running as mod_php Apache module on the server?

Please try:
- Uninstall AIOM
- Check that the folder "aiom" in "site/assets/" has been removed, if not delete manually.
- Set the rights in the "site/config.php" for $config->chmodDir to 0777 and for $config->chmodFile to 0666
- Install AIOM from current dev-branch (https://github.com/FlipZoomMedia/ProcessWire-AIOM-All-In-One-Minify/tree/dev)
